src/Core/Content/Flow/Dispatching/Action/AddCustomerAffiliateAndCampaignCodeAction.php

```diff
@@ -12,7 +12,7 @@
 use Shopware\Core\Framework\Uuid\Uuid;
 
 /**
- * @internal
```

Comment:
All the actions where actually intended to be `@internal`, you should not rely on any single concrete action, the abstract `FlowAction` is public and that should be the area where we keep backwards compatibility